docusaurus/docs/adding-typescript.md

```diff
@@ -27,7 +27,7 @@ npm install --save typescript @types/node @types/react @types/react-dom @types/j
 yarn add typescript @types/node @types/react @types/react-dom @types/jest
 ```
 
-Next, rename any file to be a TypeScript file (e.g. `src/index.js` to `src/index.tsx`) and **restart your development server**!
```

Comment:
Isn't this correct already? I mean, from the text on the same line it talks about `TypeScript`, I suppose it is correct.